docs/components/QualityChecklist.js

```diff
@@ -32,17 +32,15 @@ export default function QualityChecklist({ component }: Props): Node {
             <col style={{ width: '20%' }} />
             <col style={{ width: '60%' }} />
           </colgroup>
-          <Box display="visuallyHidden">
```

Comment:
@AlbertCarreras this is the root of the issue - could you take a look at this diff?

Reply:
If this fixes Hydration issues... gonna make you a Saint in Spain, Saint Vuerings